code/modules/emotes/emote_mob.dm

```diff
@@ -1,8 +1,17 @@
 /mob/proc/can_emote(var/emote_type)
+	. = check_mob_can_emote(emote_type)
+	if(!.)
+		to_chat(src, SPAN_WARNING("You cannot currently [emote_type == AUDIBLE_MESSAGE ? "audibly" : "visually"] emote!"))
+
+/mob/proc/check_mob_can_emote(var/emote_type)
+	SHOULD_CALL_PARENT(TRUE)
 	return (stat == CONSCIOUS)
 
-/mob/living/can_emote(var/emote_type)
-	return (..() && !(HAS_STATUS(src, STAT_SILENCE) && emote_type == AUDIBLE_MESSAGE))
+/mob/living/check_mob_can_emote(var/emote_type)
+	return ..() && !(HAS_STATUS(src, STAT_SILENCE) && emote_type == AUDIBLE_MESSAGE)
+
+/mob/living/carbon/brain/check_mob_can_emote(var/emote_type)
+	return ..() && istype(loc, /obj/item/mmi)
```

Comment:
I kind of wonder if a better approach would be to just check isSynthetic on the synth emotes. That way, it'd work for posibrains *and* MMIs. The only issue is that currently isSynthetic only returns true for brains in a posibrain or digital MMI, but since isSynthetic is supposed to return true for FBPs, saying an MMI is synthetic seems fine.